src/index.html

```diff
@@ -20,8 +20,383 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a
+            class="header__logo-link"
+            href="#"
+          >
+            <img
+              class="header__logo-img"
+              src="images/logo.png"
+              alt="Logo"
```

Comment:
For logos, use the format: "[Organization name] logo." 
Example:

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/fbacbd79-735b-47e1-984a-58b5f9495bda)